app/scripts/modules/kubernetes/src/v2/manifest/ManifestResources.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { get } from 'lodash';
+
+export interface IManifestResources {
+  cpu?: string;
+  memory?: string;
+}
+
+export interface IManifestResourceProps {
+  manifest?: {
```

Comment:
Can you add a TODO (for me) with a link to this issue here: https://github.com/spinnaker/spinnaker/issues/3239?

Reply:
done